libp2p/protocols/pubsub/gossipsub/behavior.nim

```diff
@@ -106,10 +106,10 @@ proc handleGraft*(g: GossipSub,
     let topic = graft.topicId
     trace "peer grafted topic", peer, topic
 
-    # It is an error to GRAFT on a explicit peer
+    # It is an error to GRAFT on a direct peer
     if peer.peerId in g.parameters.directPeers:
       # receiving a graft from a direct peer should yield a more prominent warning (protocol violation)
-      warn "an explicit peer attempted to graft us, peering agreements should be reciprocal",
+      warn "a direct peer attempted to graft us, peering agreements should be reciprocal",
```

Comment:
this can cause a log spam issue if they keep doing this

Reply:
I think we are supposed to trust them somewhat
The other solution would be to remove them from the trusted set in that case
(descoring isn't an option as we will never kick them due to bad score)

Reply:
hm, right about the trust - let's make a comment about it